paddle/fluid/operators/ops_extra_info.h

```diff
@@ -108,7 +108,6 @@ const std::unordered_map<std::string, ExtraAttrPropertySet>
         {"fused_transpose_X", ExtraAttrProperty::ONEDNN},
         {"fused_transpose_Y", ExtraAttrProperty::ONEDNN},
         {"mkldnn_data_type", ExtraAttrProperty::ONEDNN},
-        {"ResidualData", ExtraAttrProperty::ONEDNN},
```

Comment:
I think we could also delete `{"Bias", ExtraAttrProperty::ONEDNN},` because it's also Input, not attribute